packages/terra-date-input/src/DateInput.jsx

```diff
@@ -280,7 +280,7 @@ class DateInput extends React.Component {
     }
 
     // prevent + and - characters from being entered into number input on keyDown
-    if (event.keyCode === DateInputUtil.keyCode.KEY_MINUS || event.keyCode === DateInputUtil.keyCode.KEY_PLUS) {
+    if (event.keyCode === KeyCode.KEY_EQUALS || event.keyCode === KeyCode.KEY_DASH) {
```

Comment:
these keys don't seem to match the previous ones 👀

Reply:
At first i thought there might be a bug here but I think this is ok. Dash and minus are equivalent (keycode 189) and Equals is equivalent to plus as well when you factor in shift (keycode 187). We don't really have to take shift into account here because it doesn't hurt to prevent = as well as +.

Reply:
https://github.com/kabirbaidhya/keycode-js/blob/master/dist/keycode.cjs.js#L112-L114

Reply:
ah, thanks for clearing that up! 😅